dashboard/app/controllers/home_controller.rb

```diff
@@ -42,7 +42,27 @@ def index
       @recent_courses = recent_courses.slice(0, 2)
 
       if current_user.teacher?
-        @sections = []
+        base_url = CDO.code_org_url('/teacher-dashboard#/sections/')
+        @sections = current_user.sections.map do |section|
+          if section.script_id
+            course_name = Script.find_by_id(section.script_id)[:name]
+            course = data_t_suffix('script.name', course_name, 'title')
+            link_to_course = script_url(section.script_id)
+          else
+            course = ""
+            link_to_course = base_url
```

Comment:
What happens on the client when a section is not assigned to a course/script? Why do we need these two fields for this case, but not need the other fields?